libc/src/__support/FPUtil/BasicOperations.h

```diff
@@ -58,6 +58,130 @@ LIBC_INLINE T fmax(T x, T y) {
   }
 }
 
+template <typename T, cpp::enable_if_t<cpp::is_floating_point_v<T>, int> = 0>
+LIBC_INLINE T fmaximum(T x, T y) {
+  FPBits<T> bitx(x), bity(y);
+
+  if (bitx.is_nan()) {
+    return x;
+  } else if (bity.is_nan()) {
```

Comment:
Please use the form:

```
if (bitx. ...)
  return ...
if (bity. ...)
  return
if (bitx.sign ...)
  return
return x > y ? x : y;
```

Here and below.

Reply:
ok I will fix it. is it ok after I make my changes to do 

```
$ git add
$ git commit --amend
$ git push --force
```
So I can keep it at two commits?

Reply:
It's ok for you to keep multiple commits, don't need to `--amend`.